rules/windows/persistence_user_account_creation_event_logs.toml

```diff
@@ -17,18 +19,18 @@ false_positives = [
     """,
 ]
 index = ["winlogbeat-*", "logs-system.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
-name = "Creation of a local user account"
+name = "Windows User Account Creation"
 risk_score = 21
 rule_id = "38e17753-f581-4644-84da-0d60a8318694"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
```

Comment:
We may want to maintain it as KQL as we don't support converting types AFAIK

Reply:
switched back to kql, while testing rule didn't trigger, had to add `event.module : "system"` probably any rule using event.module in the kql query and `logs-system*` in index should be tested.